dist/components/progress/_progress.scss

```diff
@@ -0,0 +1,116 @@
+// Progress
+// ========
+//
+// Helps show users where they exist within a multiple-page workflow, such as a
+// checkout flow.
+
+// Configurable Variables
+// ---
+
+$progress__item-bg-color: hsl(0, 0%, 95%) !default;
+$progress__item-padding: if(variable-exists(v-space), $v-space, 10px) !default;
+$progress__stage-layer: if(variable-exists(stage-layer), $stage-layer, 10) !default;
+$progress__current-bg-color: darken($progress__item-bg-color, 10%) !default;
+$progress__completed-bg-color: darken($progress__item-bg-color, 20%) !default;
+
+// Arrow Divider Variables
+// 1. These two variables together determine the angle of the arrow dividers
+
+$progress__divider-angle: 90px !default; // 1
+$progress__divider-width: 20px !default; // 1
+$progress__divider-border-thickness: 2px !default;
+$progress__divider-border-color: hsl(360, 100%, 100%) !default;
+
```

Comment:
These are great configurable vars: the CSS for building this is complex, so config vars make if very easy to customize.